util/templates/defaults.yaml

```diff
@@ -310,14 +310,14 @@ params:
       en: Power limit for grid charging.
       de: Leistungslimit für Netzladung.
     default: 100
-  - name: cascade
+  - name: sun2000cascade
```

Comment:
Der Parameter gehört nciht in die defaults.yaml sondern direkt in das Template. Naming war eher die Frage ob "cascade" oder sowas wie "resetaccharge"?

Reply:
Aber kommt aus der defaults.yaml nicht die Betitelung und Beschreibung für das Config-UI auf der Weboberfläche? deshalb habe ich den Parameter mit Beschreibung dort eingefügt

Reply:
Kann, muss aber nicht. Gleiche Syntax hast Du auch in den einzelnen Templates. In die defaults kommt es nur wenns Wiederverwendung gibt.